### ISSUE TITLE
FIGURE ERROR GUI

### ISSUE THREAD

**Issue by capitanevs** [oasys-elettra-kit/OASYS1-Wiser#77]
What does  Scaling do?
![immagine](https://user-images.githubusercontent.com/13431660/150799482-8ffb2158-0e1b-411b-83d9-2e65ec67b584.png)

I expect in the previous image to change the Y on the chart, but it does not.

Similarly for "Height Unit"

**Comment by aljosahafner** [comment]
I see, you're right, this has to be fixed.

**Comment by capitanevs** [comment]
Keep in mind
![immagine](https://user-images.githubusercontent.com/13431660/150807803-07365bab-38f8-4b10-a5de-692bd1ac5ba9.png)

**Comment by aljosahafner** [comment]
Of course!

**Comment by capitanevs** [comment]
YEah yeah, it was for "personal" notice:) I mean, always get confused on what the meaning of X/YScaling is supposed to be
(if the scaling of the written data, or if the factor I have to multiply once I have read them to make them in the SI system; apparently it is the first one.  :)

**Comment by aljosahafner** [comment]
Bug fixed in `0.3.19`. Please close when tested.

**Comment by capitanevs** [comment]
![immagine](https://user-images.githubusercontent.com/13431660/155118820-81cd1d86-ba99-470c-a178-87aba6024362.png)

**Comment by capitanevs** [comment]
The figure error is read correctly (x>mm, y>nm)

However the intensity on the element has changed?

No figure error
![immagine](https://user-images.githubusercontent.com/13431660/155119192-dc497d5e-427c-424f-9dd9-b623b3717558.png)

With figure error

![immagine](https://user-images.githubusercontent.com/13431660/155119222-6d3865e2-e050-4db9-97c3-122ae70b0b6c.png)

*For sure* before the update (if the good numbers were given) this was not happening.

**Comment by aljosahafner** [comment]
Ok, weird, let's discuss today.

**Comment by capitanevs** [comment]
A Problem could still be that the x and y are not returned properly (because of the scaling factor)

if the figure error is too high,then we are deformating the mirror a lot, and we do not know what is happening to the scattered light.

i messed on a similar problem my as well:) maybe it is the same.

**Comment by capitanevs** [comment]
This error is still persistent and it has nothing to do with the issue related to "ignore" function.

Do we have any clue?

[LDM simple.ows.txt](https://github.com/oasys-elettra-kit/OASYS1-Wiser/files/8223924/LDM.simple.ows.txt)
[ldm_kbh_(mm,nm).txt](https://github.com/oasys-elettra-kit/OASYS1-Wiser/files/8223925/ldm_kbh_.mm.nm.txt)

**Comment by capitanevs** [comment]
Sorry for the inconvenience. Now it works and it works nicely!

![immagine](https://user-images.githubusercontent.com/13431660/159248417-bdfeea99-dc71-42bc-af51-9850ef1c1cd3.png)
![immagine](https://user-images.githubusercontent.com/13431660/159248490-a95dc443-611d-4c7f-a67d-53b560979396.png)
![immagine](https://user-images.githubusercontent.com/13431660/159248659-7d256efe-2231-4c29-b510-828e397163dd.png)
